tests/whitebox/integration/test_stratis.py

```diff
@@ -47,6 +53,27 @@ def testStratisRedundancy(self):
         command_line = self._MENU + ["redundancy"]
         RUNNER(command_line)
 
+    def testLowTimeout(self):
+        """
+        Getting version with low timeout variable should fail.
+        """
+        environ["STRATIS_DBUS_TIMEOUT"] = "0"
+        command_line = self._MENU + ["version"]
+
+        with self.assertRaises(StratisCliActionError) as context:
```

Comment:
Now that we've worked out what is going on, I think you're in a good position to use ```check_error```.